README.adoc

```diff
@@ -1,3 +1,7 @@
+image:https://github.com/couchbase/docs-sdk-go/actions/workflows/build.yml/badge.svg[https://github.com/couchbase/docs-sdk-go/actions/workflows/build.yml] 
+image:https://github.com/couchbase/docs-sdk-go/actions/workflows/test-ga.yml/badge.svg[https://github.com/couchbase/docs-sdk-go/actions/workflows/test-ga.yml]
+image:https://github.com/couchbase/docs-sdk-go/actions/workflows/test-dev.yml/badge.svg[https://github.com/couchbase/docs-sdk-go/actions/workflows/test-dev.yml]
```

Comment:
`test-dev` will only run on the default branch, so the badge will show a broken link for now until this is merged.